addons/sourcemod/scripting/sf2/npc/npc_chaser_pathing.sp

```diff
@@ -427,7 +427,7 @@ void SlenderChaseBossProcessMovement(int bossEnt)
 		int pitch = combatChar.LookupPoseParameter("body_pitch");
 		int yaw = combatChar.LookupPoseParameter("body_yaw");
 		float dir[3], ang[3], npcCenter[3], lookPos[3];
-		combatChar.WorldSpaceCenter(npcCenter);
+		SF2_GetBossEyePosition(bossIndex, npcCenter);
```

Comment:
Use NPCGetEyePosition instead of SF2_GetBossEyePosition, the latter is a native.